apps/cowswap-frontend/src/modules/swap/pure/warnings.tsx

```diff
@@ -101,7 +101,7 @@ export const SwapWarningsTop = React.memo(function (props: SwapWarningsTopProps)
         priceImpact={priceImpact}
         buyingFiatAmount={buyingFiatAmount}
         tradeUrlParams={tradeUrlParams}
-        sellAmount={trade?.inputAmount.toFixed(trade?.inputAmount?.currency?.decimals || 18)}
```

Comment:
@alfetopito do you remember why `toFixed()` was used instead of `toExact()`?

Reply:
Probably a mistake